tensorboard/webapp/util/dom.ts

```diff
@@ -21,3 +21,20 @@ export enum MouseEventButtons {
   FOURTH = 0b1000, // often 'back' button, but can differ by mouse controller.
   FIFTH = 0b100000, // often 'forward' button, but can differ by mouse controller.
 }
+
+let currElementId = 0;
+
+/**
+ * An opaque id intended to refer to exactly one specific DOM Element.
+ */
+export type ElementId = number;
```

Comment:
Will not mention this more but consider using `Symbol` if you want it to be opaque and want to be able to do `!elementId` to check nullness instead of `elementId ?? null` which weakly leaks the fact that `ElementId` is a number.

Reply:
Done as Symbol.

Still using an incrementing number inside the symbol, so that when printed in the console, it's easier to glance at things like:
entered [Symbol(1), Symbol(2)], exited [Symbol(3)]